### PR TITLE
Handling FHIR deps

### PR DESCRIPTION
**Pull request by ngfreiter**
This is 2 of 3 PRs that add functionality to handle FHIR dependencies specified by the user. The other PRs are:
shr-cli: https://github.com/standardhealth/shr-cli/pull/228
shr-fhir-export: https://github.com/standardhealth/shr-fhir-export/pull/166

### CODE REVIEW

File: lib/expand.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ function setLogger(bunyanLogger) {
   rootLogger = logger = bunyanLogger;
 }
 
-function expand(specifications, ...exporters) {
-  const expander = new Expander(specifications, ...exporters);
+function expand(specifications, configuration, ...exporters) {
```
Comment (cmoesel) [inline lib/expand.js:11]: I think there are other modules (e.g., `shr-es6-export`, `shr-json-schema-export`) that also use `shr-expand` -- and likely need to be updated to support this signature change.  I think it's mainly in their tests.

**Review by cmoesel — CHANGES_REQUESTED** [review]
This change affects some other modules which will also need to be updated.

File: lib/expand.js (hunk 1 above)
Comment (ngfreiter) [reply lib/expand.js:11]: You are correct, this broke the changes in `shr-es6-export`. I added a PR here https://github.com/standardhealth/shr-es6-export/pull/61 to fix that.

**Review by cmoesel — APPROVED** [review]
Looks good.  Tested by adding a dependency on the QICore IG and it worked well!  Thanks!

**Pull request merged**